src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -88,6 +78,13 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
   const absl::flat_hash_map<NodeID, std::shared_ptr<SchedulingResources>>
       &GetClusterResources() const;
 
+  /// Update resources of a node
+  void UpdateResources(const NodeID &node_id,
+                       absl::flat_hash_map<std::string, double> changed_resources);
```

Comment:
```suggestion
                       absl::flat_hash_map<std::string, double> &changed_resources);
```
Any reason why we pass the copy?